lib/core/collections.toit

```diff
@@ -2941,8 +2941,13 @@ class IdentityMap extends Map:
     return identical key key_or_probe
 
 /**
-A collection where you can add to the end, and remove items from
-  either end efficiently.
+A double-ended queue.
+
+Supports adding and removing at the beginning and the end. Both operations
```

Comment:
You can only add at one end at the moment though.

Reply:
done.